tests/printer/expr/expected/list.res.txt

```diff
@@ -2,6 +2,7 @@ let x = list{}
 let x = list{1}
 let x = list{1, 2}
 let x = list{1, 2, 3}
+let x = Belt.List.concatMany([list{1, 2, ...x}, list{3, ...x}])
 
```

Comment:
could this be improved? print it as `list{ 1, 2, ...x , 3 , ...x}`

Reply:
It could be, but just by changing the printer can cause some problems. For example, list{. . x, . .y} and Belt.List.concatMany([x, y]) will be printed as the same thing. I am still trying to solve this problem.

Reply:
can you add an attribute to Belt.List.concatMany to avoid accidental simplifications?

Reply:
This is what is done for string templates, I believe, to differentiate from normal string concatenation.

Reply:
Check what `isTemplateLiteral` does.

Reply:
Thank you for all your suggestions, I'll learn more about that.